### PR TITLE
net-p2p/go-ethereum: Version bump to 1.8.2

### PR DESCRIPTION
**Pull request by MathyV**
Closes: https://bugs.gentoo.org/647992
Closes: https://bugs.gentoo.org/648030
Package-Manager: Portage-2.3.19, Repoman-2.3.6

### CODE REVIEW

File: net-p2p/go-ethereum/Manifest [hunk 1]
```diff
@@ -1,2 +1 @@
-DIST go-ethereum-1.7.2.tar.gz 7037907 BLAKE2B eb1e358e42c743a931995e7ee46dc2441356159fc9095e8a886cc6a0495cb2d2e77ee7df4c6dd98bc790397819fc43cdedba6cf28de1e4f3dcdc4dd71106f8e5 SHA512 7bc5957f1523fa5ec67650c4305633aa4c6098a243ef415f9ef56a4e4763eb2ef9f69514ed15810fcdb75f11e87cbd27c4f2af802c32a03be91b37a7b08ea520
```
Comment (mgorny) [inline net-p2p/go-ethereum/Manifest]: Please split this into two commits: one adding the new version, the other removing old.

**Comment by mgorny** [comment]
Ping.

**Comment by MathyV** [comment]
Done and updated to new version at the same time

File: net-p2p/go-ethereum/go-ethereum-1.8.2.ebuild [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2017 Gentoo Foundation
```
Comment (mgorny) [inline net-p2p/go-ethereum/go-ethereum-1.8.2.ebuild]: But… this is still combining removal with adding new version?

Comment (MathyV) [reply net-p2p/go-ethereum/go-ethereum-1.8.2.ebuild]: I thought you meant only 1.7.2 needed to be removed separately, changed.

**Comment by mgorny** [comment]
```
--- FAIL: TestPathExpansion (0.00s)
	customflags_test.go:38: test ~/tmp, got /tmp/portage/net-p2p/go-ethereum-1.8.2/homedir/tmp, expected /var/tmp/portage/tmp
FAIL
FAIL	github.com/ethereum/go-ethereum/cmd/utils	0.074s
```

That is clearly a bug in tests, so I'm merging it anyway. Thanks!

**Pull request closed without merging**